src/utils/grid.css

```diff
@@ -1,247 +1,49 @@
-.u-grid-1\/12 {
-  grid-column: span 1;
-}
-
-.u-grid-2\/12 {
-  grid-column: span 2;
-}
-
-.u-grid-3\/12 {
-  grid-column: span 3;
-}
-
-.u-grid-4\/12 {
-  grid-column: span 4;
-}
-
-.u-grid-5\/12 {
-  grid-column: span 5;
-}
-
-.u-grid-6\/12 {
-  grid-column: span 6;
-}
-
-.u-grid-7\/12 {
-  grid-column: span 7;
-}
-
-.u-grid-8\/12 {
-  grid-column: span 8;
-}
-
-.u-grid-9\/12 {
-  grid-column: span 9;
-}
-
-.u-grid-10\/12 {
-  grid-column: span 10;
-}
-
-.u-grid-11\/12 {
-  grid-column: span 11;
-}
-
-.u-grid-12\/12 {
-  grid-column: span 12;
-}
-
-@media (min-width: 25rem) {
-  .u-grid-1\/12\@sm {
-    grid-column: span 1;
-  }
-
-  .u-grid-2\/12\@sm {
-    grid-column: span 2;
-  }
-
-  .u-grid-3\/12\@sm {
-    grid-column: span 3;
-  }
-
-  .u-grid-4\/12\@sm {
-    grid-column: span 4;
-  }
-
-  .u-grid-5\/12\@sm {
-    grid-column: span 5;
-  }
-
-  .u-grid-6\/12\@sm {
-    grid-column: span 6;
-  }
-
-  .u-grid-7\/12\@sm {
-    grid-column: span 7;
-  }
-
-  .u-grid-8\/12\@sm {
-    grid-column: span 8;
-  }
-
-  .u-grid-9\/12\@sm {
-    grid-column: span 9;
-  }
-
-  .u-grid-10\/12\@sm {
-    grid-column: span 10;
-  }
-
-  .u-grid-11\/12\@sm {
-    grid-column: span 11;
-  }
-
-  .u-grid-12\/12\@sm {
-    grid-column: span 12;
-  }
-}
-
-@media (min-width: 40rem) {
-  .u-grid-1\/12\@md {
-    grid-column: span 1;
-  }
-
-  .u-grid-2\/12\@md {
-    grid-column: span 2;
-  }
-
-  .u-grid-3\/12\@md {
-    grid-column: span 3;
-  }
-
-  .u-grid-4\/12\@md {
-    grid-column: span 4;
-  }
-
-  .u-grid-5\/12\@md {
-    grid-column: span 5;
-  }
-
-  .u-grid-6\/12\@md {
-    grid-column: span 6;
-  }
-
-  .u-grid-7\/12\@md {
-    grid-column: span 7;
-  }
-
-  .u-grid-8\/12\@md {
-    grid-column: span 8;
-  }
-
-  .u-grid-9\/12\@md {
-    grid-column: span 9;
-  }
-
-  .u-grid-10\/12\@md {
-    grid-column: span 10;
-  }
-
-  .u-grid-11\/12\@md {
-    grid-column: span 11;
-  }
-
-  .u-grid-12\/12\@md {
-    grid-column: span 12;
-  }
-}
-
-@media (min-width: 60rem) {
-  .u-grid-1\/12\@lg {
-    grid-column: span 1;
-  }
-
-  .u-grid-2\/12\@lg {
-    grid-column: span 2;
-  }
-
-  .u-grid-3\/12\@lg {
-    grid-column: span 3;
-  }
-
-  .u-grid-4\/12\@lg {
-    grid-column: span 4;
-  }
-
-  .u-grid-5\/12\@lg {
-    grid-column: span 5;
-  }
-
-  .u-grid-6\/12\@lg {
-    grid-column: span 6;
-  }
-
-  .u-grid-7\/12\@lg {
-    grid-column: span 7;
-  }
-
-  .u-grid-8\/12\@lg {
-    grid-column: span 8;
-  }
-
-  .u-grid-9\/12\@lg {
-    grid-column: span 9;
-  }
-
-  .u-grid-10\/12\@lg {
-    grid-column: span 10;
-  }
-
-  .u-grid-11\/12\@lg {
-    grid-column: span 11;
-  }
-
-  .u-grid-12\/12\@lg {
-    grid-column: span 12;
-  }
-}
-
-@media (min-width: 75rem) {
-  .u-grid-1\/12\@xl {
+@responsive {
+  .u-grid-1\/12 {
```

Comment:
Do we want to lose the `/12` part right away here?

Reply:
Yes removed them 👍